.mergify.yml

```diff
@@ -60,13 +60,9 @@ pull_request_rules:
           - status-success=build & deploy docs
         - or:
           - -files~=(\.rs|Cargo\.toml|Cargo\.lock|\.github/scripts/cargo-clippy-before-script\.sh|\.github/workflows/cargo\.yml)$
-          - and:
```

Comment:
according to https://docs.mergify.com/configuration/conditions/, i think we can also remove this ` and:` because it would only have a single `or`.